OracleManagementAgent/README.md

```diff
@@ -112,50 +134,105 @@ Users can provide custom shell script commands to execute before starting Manage
+<!-- markdownlint-disable MD046 -->
+    * Once a Management Agent instance is deregistered that instance must be shutdown and any associated state files must be removed from the filesystem. Starting a deregistered Management Agent instance again can result in unregistered agent failures. This situation can present itself when old state files from a prior installation are present on the filesystem and made available to a new Management Agent container deployment. Run the command given below on the host filesystem to perform the necessary cleanup on the bind mount location and perform the deployment again starting at the prerequisites step.
+
+    ```shell
+    # cleanup old files from prior installation
+    $ rm -rf /home/$USER/oracle/
+    ```
+<!-- markdownlint-enable MD046 -->
+<!-- markdownlint-enable MD013 -->
+
+## Helpful commands
 
 1. Starting a stopped Management Agent Container
 
     ```shell
+    # start container and agent
     $ docker start mgmtagent-container
     ```
 
 1. Stopping a running Management Agent Container
 
     ```shell
+    # stop container and agent
     $ docker stop mgmtagent-container
     ```
 
 1. Inspecting logs of Management Agent Container
 
     ```shell
+    # inspect container and agent install logs
     $ docker logs mgmtagent-container
     ```
 
-1. Cleanup volume using Docker Compose
+1. Gathering UID and GID of a user from the host environment
 
     ```shell
-    $ docker-compose down --volumes
+    # see creating a user account below
+    $ id -u <username> # prints UID of user
+    $ id -g <username> # prints GID of user
     ```
 
-1. Cleanup volume using Docker CLI
+1. Creating a named user account during image development (optional)
 
     ```shell
-    $ docker volume rm mgmtagent-volume
+    # Commands given below can use used to precreate a user account in container image
+
+    $ groupadd -g <numeric-gid-value> <desired-groupname> # create group with gid
+    # example:
+    $ groupadd -g 9100 agentadmingrp
+
+    $ useradd <desired-username> -u <numeric-uid-value> -g <numeric-gid-value> -m -s /bin/bash # create user with uid/gid
+    # example:
+    $ useradd agentadminusr -u 9200 -g 9100 -m -s /bin/bash
+
+    # Note: Add useradd/groupadd to Dockerfile prefixed with the RUN directive to create user during image development
     ```
 
+## Container Files for Older Releases
+
+The Oracle Management Agent older container files mentioned below are no longer actively maintained and they are kept in this repository for historical purposes only.
+
+* Oracle Management Agent container files version 1.0.0 [`docker-images/OracleManagementAgent/dockerfiles/1.0.0`](./dockerfiles/1.0.0)
+
+**Notes:**
+
+* Oracle Management Agent container files version 1.0.0 require elevated privileges to run and therefore are not compatible with the latest version found in this repository. Upgrading a version 1.0.0 container to run with the latest container files is also not supported for the same reason.
```

Comment:
You don't need to keep these for historical purposes: Git does that for you automatically as the files will be in the git history. Removing them from the current state would reduce confusion and the likelihood of anyone using them inadvertantly.

Reply:
The latest scripts runs container and Management Agent in a secure mode where the container entrypoint runs with user specified user id and group id instead of the previous default root user. This is not compatible with v1.0.0 scripts and some of our users have already started using v1.0.0 of the scripts and we want to give them some time to migrate their containers to the latest, in future we will remove the 1.0.0 dir.

Reply:
As mentioned, GitHub still allows access to previous commits. Removing the content now means we greatly decrease the chance of anyone who hasn't yet built this image from picking that option. Leaving it in place just increases the amount of folks who will use the image that requires privilege escalation.